src/index.js

```diff
@@ -0,0 +1,142 @@
+const state = {
+  currentTemp: 80,
+  lat: 40.594,
+  lon: -74.6049,
+};
+
+const tempConvert = function (K) {
+  F = Math.round(((K - 273.15) * 9) / 5 + 32);
+
+  return F;
+};
+
+const setCurrentTemp = function (temp) {
+  state.currentTemp = tempConvert(temp);
+};
+
+const tempColor = function (temperature) {
+  if (temperature >= 80) {
+    return 'text-color-red';
+  } else if (temperature >= 70) {
+    return 'text-color-orange';
+  } else if (temperature >= 60) {
+    return 'text-color-skyblue';
+  } else if (temperature >= 50) {
+    return 'text-color-green';
+  } else {
+    return 'text-color-plum';
+  }
+};
+
+const tempPic = function (temperature) {
+  if (temperature >= 80) {
+    return 'red-sand.jpeg';
+  } else if (temperature >= 70) {
+    return 'orange-park.jpeg';
+  } else if (temperature >= 60) {
+    return 'summer-beach.jpeg';
+  } else if (temperature >= 50) {
+    return 'bow-lake.jpeg';
+  } else {
+    return 'sunset.jpeg';
+  }
+};
+
+const displayTemperature = function (temperature) {
+  const tempDisplay = document.getElementById('tempDisplay');
+  const landScape = document.getElementById('landScape');
+  const color = tempColor(temperature);
+  const imgFileName = tempPic(temperature);
+  const displayString = `<p class="${color}">Current Temperature: ${temperature} F</p>`;
+  const imgFileNameString = `<img alt="Nice Landscape" src="assets/${imgFileName}">`;
```

Comment:
Let's let the html file create these elements instead of us. I think we should also change `color` and `temperature` separately instead of keeping them both in one place.

Refer to my comment in the HTML to get a full view of the suggestion changes we could make:

```suggestion
  tempDisplay.value = temperature;
  tempDisplay.className = color;
  landscape.src = `assets/${imgFileName}`;
```